code/game/machinery/doors/windowdoor.dm

```diff
@@ -187,16 +178,15 @@
 			qdel(src)
 		return TRUE
 
-	if(!isliving(I))
-		if(I.iscrowbar() && user.a_intent == I_HELP)
-			if(inoperable())
-				visible_message("\The [user] forces \the [src] [density ? "open" : "closed"].")
-				if(density)
-					open(1)
-				else
-					close(1)
+	if(isobj(I) && I.iscrowbar() && user.a_intent == I_HELP)
```

Comment:
Whyyy do we need `isobj()` for attackby, which is always `obj/item` ??

Reply:
![image](https://user-images.githubusercontent.com/38594443/172015837-f0873f52-3332-44eb-a1b4-2d0a2a3d3cdd.png)
![iknownow](https://user-images.githubusercontent.com/38594443/172015863-00b58a12-520b-4b29-8d3c-71b3b8df4f6f.jpg)